install_config/topics/manually_configuring_for_gce-proc.adoc

```diff
@@ -0,0 +1,162 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Manually configuring {product-title} for GCE
+
+== Manually configuring master hosts for GCE
+
+Perform the following on all master hosts.
+
+.Procedure
+
+. Edit the master configuration file at `/etc/origin/master/master-config.yaml` by
+default on all masters and update the contents of the `apiServerArguments` and
+`controllerArguments` sections:
++
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+  controllerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+----
++
+[IMPORTANT]
+====
+When triggering a containerized installation, only the *_/etc/origin_* and
+*_/var/lib/origin_* directories are mounted to the master and node container.
+Therefore, ensure *_master-config.yaml_* is in the *_/etc/origin/master_*
+directory instead of *_/etc/_*.
+====
+
+. When you configure {product-title} for GCE using Ansible, the `/etc/origin/cloudprovider/gce.conf` file is created automatically. Because you are manually configuring {product-title} for GCE, you must create the file:
++
+[subs=+quotes]
+----
+[Global]
+project-id = <project-id> <1>
+network-name = <network-name> <2>
+node-tags = <node-tags> <3>
+node-instance-prefix = <instance-prefix> <4>
+multizone = true <5>
+----
+<1> Set to the GCE project ID where the existing instances are running.
+<2> Set to the network name if not using the default.
+<3> 
```

Comment:
For running load balancer services using the Google compute platform, the nodes (Compute Engine VM instances) must be tagged with a special GCP network tag to work.
The tag should be <node-instance-prefix>ocp (add ocp suffix). For example, if the value of `node-instance-prefix` parameter is set to mycluster, the nodes must be tagged with myclusterocp.
See [Adding and Removing Network Tags](https://cloud.google.com/vpc/docs/add-remove-network-tags) for more information on how to add network tags to Compute Engine VM instances.